### PR TITLE
Updated to 2.2

### PR DESCRIPTION
**Pull request by RCdiy**
Updated to 2.2

### CODE REVIEW

File: companion-introduction.md [hunk 1]
```diff
@@ -107,59 +109,176 @@ Simulator settings:
...
-* Profiles: Allows storing different setting sets and easily switching between them. For example, if you have 2 different radios with different firmwares or board types it is not convenient to have to redo all settings (firmware selection, ticking options,...) every time you want to do operations on the other radio. So you can configure all settings, choose an empty profile with the number box, type a name to identify the particular radio, and click save. Do the same for the second radio. You will now be able to select the correct profile for the radio you are about to work on with the profile selector button and menu entry on OpenTX companion's main window. Note that the profiles can also store and retrieve each radio's stick calibration and hardware settings (voltage alarms, audio modes,...) from the General Settings page of an open document. This allows copying a document from one radio to the other without needing recalibration or reentering the hardware settings.
+Several model selection windows can be open at the same time. Models and settings can be coppied between these wondows.
 
-Joystick: This lets you configure a joystick to simulate the sticks in the radio simulator.
```
Comment (schwabe) [inline companion-introduction.md:113]: You removed this point without moving it anywhere else.

**Comment by schwabe** [comment]
Since I pulled in all other requests, this does not merge at all anymore. I will pull in the changes manually and fix things as I go but I can take a few days.

File: companion-introduction.md (hunk 1 above)
Comment (RCdiy) [reply companion-introduction.md:113]: Yes... There is no Joystick build option that is why... The information is still there however... Joystick is default...
**massstorage:** Connecting the USB cable when the Tx is on the microSD cards gets mounted as a drive. If this option is not selected connecting the USB cable enables using the radio as a joystick in flight simulators.
**cli:** Connecting the USB cable enables access to the Command Line Interface.

**Comment by RCdiy** [comment]
When done maybe close #36

File: companion-introduction.md (hunk 1 above)
Comment (RCdiy) [reply companion-introduction.md:113]: I think I saw a pull request for an entire separate joystick section... #32 In that request take care/be careful of the move build options section pull.... you have already noted that these merges will be done by you manually... thanks:)

**Comment by RCdiy** [comment]
@schwabe I've posted this on my website and while doing so generated a lot more image files. I'm attaching them here for you to use if you wish. Also while posting I ended up doing more edits. I decided to completely remove the section on bootloader updating and zadig, dfu-util since that's causing more confusion than needed.
[Archive.zip](https://github.com/opentx/opentx-taranis-manual/files/1219554/Archive.zip)

**Comment by RCdiy** [comment]
I've redone the companion write up considerably. If you plan on seperating it out into seperate files take a look at the link... it may help you organize the files.
http://rcdiy.ca/opentx-guide/introduction-to-opentx-companion/
Still a work in progress.

**Comment by schwabe** [comment]
@RCdiy as it stand the commit here is not mergable at all.I would not recommend working further on your branch as it would only diverge even further. I disagree with removing Zadiag instruction. THey are required for the Horus and for recovery on the Taranis.

**Comment by RCdiy** [comment]
Put zadig in advanced/trouble shooting... the idea is that most people do not need it... the problem is people are installing in and then coming to the forums with problems... that was the reason for the suggestion on changing the explanation of zadig...

**Comment by RCdiy** [comment]
I've found too many typos and changes I'd want to make... maybe close this pull request and copy from the webpage?

**Comment by schwabe** [comment]
@RCdiy I merge most of the changes you did. I hopefully did not forget anything. It is best to work from the version that is now in the repository. Otherwise merging/pull request are going to be a nightmare.

**Pull request closed without merging**